src/main/webapp/script.js

```diff
@@ -0,0 +1,100 @@
+var zoomedIn = false;
```

Comment:
One more thing!

As noted in chat; only use `let` or `const`, don't use `var`:
https://google.github.io/styleguide/jsguide.html#features-use-const-and-let